pkgs/development/libraries/quarto/default.nix

```diff
@@ -11,14 +11,18 @@
 , makeWrapper
 , python3
 , extraPythonPackages ? ps: with ps; []
+
+# Allow overriding the Python and R used by Quarto
+, QUARTO_PYTHON ? "${python3.withPackages (ps: with ps; [ jupyter ipython ] ++ (extraPythonPackages ps))}/bin/python3"
```

Comment:
Also, I wonder how this argument should interact with extraPythonPackages. If the user has a python3 in hand it may already be something wrapped, so maybe we should have some logic not to use both of these arguments?